api/config/v1alpha1/groupversion_info.go

```diff
@@ -1,7 +1,7 @@
 // Package v1alpha1 contains API Schema definitions for the config v1alpha1 API group.
 //
-//+kubebuilder:object:generate=true
-//+groupName=config.gateway.envoyproxy.io
+// +kubebuilder:object:generate=true
+// +groupName=config.gateway.envoyproxy.io
```

Comment:
This is the result of `gofmt` 1.19, which is different than the `gofmt` 1.18 that we're still using. This should be in a separate Go 1.19 PR.

Reply:
Thanks, I will remove this.